compiler/next/doc/dummy-doxygen-output/xml/combine.xslt

```diff
@@ -0,0 +1,15 @@
+<!-- XSLT script to combine the generated output into a single file. 
```

Comment:
These dummy-docxygen-output files allow `make docs` to build the compiler documentation without requiring that `doxygen` be installed. We could alternatively require `doxygen` for `make docs`.